demo/grid-event-context-demos.html

```diff
@@ -0,0 +1,65 @@
+<dom-module id="grid-event-context-demos">
+  <template>
+    <style include="vaadin-component-demo-shared-styles">
+      :host {
+        display: block;
+      }
+    </style>
+
+    <style>
+      vaadin-grid {
+        text-transform: none;
```

Comment:
The whole style block with this `text-transform` rule seems to be an old copy-paste leftover and can be removed.

Reply:
Done